File: C/exercice3_1.c
#include <stdio.h>
#include <stdlib.h>
#include <unistd.h>
#include <sys/types.h>
#include <sys/wait.h>
#include <time.h>
#include <errno.h>

int main(int argc, char const *argv[])
{
	if (argc < 3 || argc > 3)
	{
		printf("argument invalide\n") ;
		return (0) ;
	}
	int a = atoi(argv[1]) , b = atoi(argv[2]) ;
	printf("Leur somme: %d\n", a + b);
	return 0;
}

File: Python/exercice3_1.py
import sys

def main():
    
    if len(sys.argv) != 3:
        print("argument invalide")
        return 0
    
    a = int(sys.argv[1])
    b = int(sys.argv[2])
    
    print(f"Leur somme: {a + b}")
        

if __name__ == "__main__":
    main()